src/EnergyPlus/UnitarySystem.cc

```diff
@@ -0,0 +1,12162 @@
+          heatingCycRatio(0.0), heatingSpeedRatio(0.0), heatingSpeedNum(0), speedNum(0), dehumidInducedHeatingDemandRate(0.0),
+          coolCoilWaterFlowRatio(0.0), heatCoilWaterFlowRatio(0.0), fanPartLoadRatio(0.0), totalAuxElecPower(0.0), heatingAuxElecConsumption(0.0),
+          coolingAuxElecConsumption(0.0), elecPower(0.0), elecPowerConsumption(0.0), lastMode(0), firstPass(true), totCoolEnergyRate(0.0),
+          sensCoolEnergyRate(0.0), latCoolEnergyRate(0.0), totHeatEnergyRate(0.0), sensHeatEnergyRate(0.0), latHeatEnergyRate(0.0),
+          designFanVolFlowRateEMSOverrideOn(false), maxHeatAirVolFlowEMSOverrideOn(false), maxCoolAirVolFlowEMSOverrideOn(false),
+          maxNoCoolHeatAirVolFlowEMSOverrideOn(false), designFanVolFlowRateEMSOverrideValue(0.0), maxHeatAirVolFlowEMSOverrideValue(0.0),
+          maxCoolAirVolFlowEMSOverrideValue(0.0), maxNoCoolHeatAirVolFlowEMSOverrideValue(0.0), EMSOverrideSensZoneLoadRequest(false),
+          EMSOverrideMoistZoneLoadRequest(false), EMSSensibleZoneLoadValue(0.0), EMSMoistureZoneLoadValue(0.0), stageNum(0), staged(false),
+          coolCountAvail(0), coolIndexAvail(0), heatCountAvail(0), heatIndexAvail(0), heatingFanSpeedRatio(0.0), coolingFanSpeedRatio(0.0),
+          noHeatCoolSpeedRatio(0.0), myFanFlag(true), myCheckFlag(true), controlZoneMassFlowFrac(0.0), sensibleLoadMet(0.0), latentLoadMet(0.0),
+          myStagedFlag(false), sensibleLoadPredicted(0.0), moistureLoadPredicted(0.0), faultyCoilSATFlag(false), faultyCoilSATIndex(0),
+          faultyCoilSATOffset(0.0), TESOpMode(0), HXAssistedSensPLRIter(0), HXAssistedSensPLRIterIndex(0), HXAssistedSensPLRFail(0),
+          HXAssistedSensPLRFailIndex(0), HXAssistedSensPLRFail2(0), HXAssistedSensPLRFailIndex2(0), HXAssistedLatPLRIter(0),
+          HXAssistedLatPLRIterIndex(0), HXAssistedLatPLRFail(0), HXAssistedLatPLRFailIndex(0), HXAssistedCRLatPLRIter(0),
+          HXAssistedCRLatPLRIterIndex(0), HXAssistedCRLatPLRFail(0), HXAssistedCRLatPLRFailIndex(0), HXAssistedCRLatPLRFail2(0),
+          HXAssistedCRLatPLRFailIndex2(0), SensPLRIter(0), SensPLRIterIndex(0), SensPLRFail(0), SensPLRFailIndex(0), LatPLRIter(0),
+          LatPLRIterIndex(0), LatPLRFail(0), LatPLRFailIndex(0), HeatCoilSensPLRIter(0), HeatCoilSensPLRIterIndex(0), HeatCoilSensPLRFail(0),
+          HeatCoilSensPLRFailIndex(0), SuppHeatCoilSensPLRIter(0), SuppHeatCoilSensPLRIterIndex(0), SuppHeatCoilSensPLRFail(0),
+          SuppHeatCoilSensPLRFailIndex(0), DXCoilSensPLRIter(0), DXCoilSensPLRIterIndex(0), DXCoilSensPLRFail(0), DXCoilSensPLRFailIndex(0),
+          MSpdSensPLRIter(0), MSpdSensPLRIterIndex(0), MSpdCycSensPLRIter(0), MSpdCycSensPLRIterIndex(0), MSpdLatPLRIter(0), MSpdLatPLRIterIndex(0),
+          MSpdCycLatPLRIter(0), MSpdCycLatPLRIterIndex(0), LatMaxIterIndex(0), LatRegulaFalsIFailedIndex(0), lowSpeedCoolFanRatio(0.0),
+          lowSpeedHeatFanRatio(0.0)
```

Comment:
This is one long initialization  list. Can any of these be packaged up in structs? Looks likes a lot of these could be grouped into structs.

Reply:
This is one of the things I don't get. I don't know what it means to package into structs. I'll work on this later and will need your guidance to get this right.

Reply:
@rraustad I'm seeing a lot of args with the same prefix (e.g. `MSpdSensPLRIter`), and for me those are things that are candidates for being put into a struct together rather than just floating around loose with the name prefix connecting them together. Does that make sense?

Reply:
Yes it does. I have been thinking about what to do here. All these ints (the *iter and *index vars) are only used for recurring warning messages, I'll be that makes you stop and think. So I could move all these to a warning struct and inherit? that in UnitarySys. Not sure how this is done but will give it a try when I get farther along.